boilerplate/frontend/angular-prebuilt/README.md

```diff
@@ -9,7 +9,7 @@ This demo app demonstrates the following use cases:
 -   Forgot password flow
```

Comment:
the boilerplates need to be updated to use the new way of adding the pre built UI